### PR TITLE
Add SELinux documentation

### PR DESCRIPTION
**Pull request by xinfengliu**
Signed-off-by: Xinfeng Liu <Xinfeng.Liu@gmail.com>

This PR adds SELinux documentation.

### CODE REVIEW

File: docs/selinux.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# SELinux
+
```
Comment (mviitane) [inline docs/selinux.md:2]: I'd like to see the page starting with a simple explanation about what is SELinux, and why someone would use it on Kubernetes. Just a few sentences/lines would be enough.

Comment (xinfengliu) [reply docs/selinux.md:2]: Thanks. I just refined the document. (most of parts of "Overview" section are from Wikipedia).

**Review by mviitane — DISMISSED** [review]
LGTM

**Comment by xinfengliu** [comment]
The latest commit `016d3bcc` is just a little rewording about "the doc is for CentOS/RHEL" (because I didn't test it on other OS distros).

**Review by mviitane — APPROVED** [review]
LGTM

**Comment by xinfengliu** [comment]
Thanks Mikko. I'm going to merge it later today if there's no other comments/objections.

**Review by jnummelin — APPROVED** [review]

**Pull request merged**